packages/load/tests/loaders/schema/schema-from-typedefs.spec.ts

```diff
@@ -189,5 +189,23 @@ describe('schema from typedefs', () => {
         includeSources: true,
       });
     })
+
+    it('should be able to load a schema with Query type but another type as a root query type', async () => {
+      const schema = await loadSchema(['./tests/loaders/schema/test-files/schema-dir/hasura/schema.graphql'], {
+        assumeValidSDL: true,
+        loaders: [new GraphQLFileLoader()],
+        sort: true,
+        convertExtensions: true,
+      });
+
+      const documents = await loadDocuments(['./tests/loaders/schema/test-files/schema-dir/hasura/attusers.graphql', './tests/schema-dir/hasura/users.graphql'], {
```

Comment:
You made a typo with the second document..! 
['./tests/loaders/schema/test-files/schema-dir/hasura/attusers.graphql', './tests/schema-dir/hasura/users.graphql']
should be
['./tests/loaders/schema/test-files/schema-dir/hasura/attusers.graphql', './tests/loaders/schema/test-files/schema-dir/hasura/users.graphql']